paimon-core/src/main/java/org/apache/paimon/table/source/DeletionFile.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.paimon.table.source;
+
+import org.apache.paimon.annotation.Public;
+import org.apache.paimon.io.DataInputView;
+import org.apache.paimon.io.DataOutputView;
+
+import javax.annotation.Nullable;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Deletion file for data file, the first 4 bytes are length, should, the following is the bitmap
+ * content.
+ *
+ * <ul>
+ *   <li>The first 4 bytes are length, should equal to {@link #length()}.
+ *   <li>Next 4 bytes are the magic number, should be equal to 1581511376.
+ *   <li>The remaining content should be a RoaringBitmap.
+ * </ul>
+ */
+@Public
+public class DeletionFile {
+
+    private final String path;
+    private final long offset;
```

Comment:
offset and length maybe can be int, because the size of deletionFile will not be too large, currently, the dv ranges in index meta are just int pair now...

Reply:
It is good to keep extensibility here.